test.js

```diff
@@ -57,6 +57,14 @@ test('removeQueryParameters option', t => {
 	t.is(m('www.sindresorhus.com?foo=bar&utm_medium=test&ref=test_ref', opts), 'http://www.sindresorhus.com/?foo=bar');
 });
 
+test('normalizeHttps option', t => {
+	const opts = {normalizeHttps: true};
+
+	t.is(m('http://sindresorhus.com'), 'http://sindresorhus.com');
```

Comment:
You're right. Other option tests also have the same lines for clarification. I should have read entire code, not just PR diff.